lib/nerves_hub/devices/alarms.ex

```diff
@@ -61,6 +62,15 @@ defmodule NervesHub.Devices.Alarms do
     |> Repo.one!()
   end
 
+  def get_current_alarms_for_device(device) do
+    device.id
+    |> Devices.get_latest_health()
+    |> case do
+      %DeviceHealth{data: data} -> data["alarms"]
```

Comment:
```suggestion
      %DeviceHealth{data: %{"alarms" => alarms}} when is_list(alarms) ->
        for {alarm, description} <- alarms, do: {String.trim_leading(alarm, "Elixir."), description}
```
I think trimming here would be useful as this would be needed anywhere in the app and not just the UI

Reply:
I agree